### ISSUE TITLE
django-allauth-2fa in pypi seems to be built for python2 only

### ISSUE THREAD

**Issue by martianrock** [valohai/django-allauth-2fa#48]
pip install from pypi fails to find suitable version if python3 is used but when installing directly from github, install (and the 2fa itself) works fine under python3.

**Comment by clokep** [comment]
As far as I know there's nothing special you need to do when pushing to pypi in order to 'enable' python 3 support.

I can reproduce your error though:
```
$ pip3 install django_allauth_2fa
Collecting django_allauth_2fa
  Could not find a version that satisfies the requirement django_allauth_2fa (from versions: )
No matching distribution found for django_allauth_2fa
```

Thanks for reporting this, I'll need to look into this more later.

**Comment by clokep** [comment]
Fixed in #50.

I'm going to push a release in a few minutes.

**Comment by clokep** [comment]
Should be all set in 0.4.4! Let me know if you have more problems. 👍
